app/views/forever_style_guide/sections/components/navigation/_nav_account_dropdown_menu.erb

```diff
@@ -19,13 +19,7 @@
         <% if current_user.superadmin? %>
           <li class="dropdown-menu-account-item">
             <%= link_to store_admin_url, class: 'dropdown-menu-nav-link dropdown-menu-nav-link-sm dropdown-menu-nav-link-no_margin', title: 'Administer The Forever Store' do %>
-              <i class='fa fa-shopping-basket fa-fw'></i>Store Admin
-            <% end %>
-          </li>
-
-          <li class="dropdown-menu-account-item">
-            <%= link_to web_app_admin_url, class: 'dropdown-menu-nav-link dropdown-menu-nav-link-sm dropdown-menu-nav-link-no_margin', title: 'Administer The Forever Web App' do %>
-              <i class='fa fa-magic fa-fw'></i>Web App Admin
+              <i class='fa fa-shopping-basket fa-fw'></i>Admin
```

Comment:
@katywatkins just curious what is the reasoning for the shopping basket being icon here?